src/Elements/WrapperStop.php

```diff
@@ -86,19 +86,19 @@ protected function compile()
 			if ($objContent->sliderSpaceBetween) $arrParams['spaceBetween'] = (int)$objContent->sliderSpaceBetween;
 			if ($objContent->sliderEffect) $arrParams['effect'] = $objContent->sliderEffect;
 			if ($objContent->sliderContinuous) $arrParams['loop'] = true;
-			if ($objContent->sliderButtons)
-			{
-				$arrParams['nextButton'] = '#swiper-'.$objContent->id.' .swiper-button-next';
-				$arrParams['prevButton'] = '#swiper-'.$objContent->id.' .swiper-button-prev';
-			}
-			if ($objContent->sliderPagination)
-			{
-				$arrParams['pagination'] = '#swiper-'.$objContent->id.' .swiper-pagination';
-				$arrParams['paginationClickable'] = true;
-			}
+            		if ($objContent->sliderButtons)
+            		{
+                		// set navigation to true, so it will be enabled
+                		$arrParams['navigation'] = true;
+            		}
+            		if ($objContent->sliderPagination)
+            		{
+                		// set pagination to true, so it will be enabled
+                		$arrParams['pagination'] = true;
+            		}
```

Comment:
Ah, can you change the indents to tabs here please?

Reply:
changed the indents.